### PR TITLE
LPD-43013 ConfigurationEntry form

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-43013

### CODE REVIEW

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListDisplayContext.java [hunk 1]
```diff
@@ -45,23 +60,64 @@
...
 		cpRequestHelper = new CPRequestHelper(httpServletRequest);
 
 		liferayPortletResponse = cpRequestHelper.getLiferayPortletResponse();
 	}
 
+	public List<CommerceAvailabilityEstimate> getCommerceAvailabilityEstimates()
+		throws PortalException {
+
+		return commerceAvailabilityEstimateService.
+			getCommerceAvailabilityEstimates(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListDisplayContext.java:97]: will this throw an exception if no avalibalitity estimates are set

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java [hunk 2]
```diff
@@ -1406,6 +1407,12 @@ public CPDefinition deleteCPDefinition(CPDefinition cpDefinition)
 			companyGroup.getGroupId(), CProduct.class,
 			cpDefinition.getCProductId());
 
+		// Commerce product configuration entries
+
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java:1411]: this is fine, but we need to also add a delete method that can't delete entries of a master configuration list

**Review by andrea-ale-sbarra — APPROVED** [review]
just a line to check.
Thank you!

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Pull request closed without merging**